TGeant3/TGeant3TGeo.cxx

```diff
@@ -518,6 +521,9 @@ extern "C" type_of_call void ggperpTGeo(Float_t *, Float_t *, Int_t &);
 Gcvol1_t *gcvol1 = 0;
 TGeoNode *gCurrentNode = 0;
 TGeant3TGeo *geant3tgeo = 0;
+TMCManager *mcManager = 0;
```

Comment:
Please, cheange the variable name to start with g (naming convention for global variables): either gMCManager or gmcManager